### PR TITLE
Update to Ember 1.11(.1!)

### PR DESCRIPTION
**Pull request by novaugust**
- remove `bind-attr` all over the place
- includes one hack for a bug in 1.11, hopefully fixed in 1.11.1

Ember 1.11 release notes: http://emberjs.com/blog/2015/03/27/ember-1-11-0-released.html

^ Pretty important read, particularly the bit on `htmlSafe` and XSS in script/style context.

### CODE REVIEW

**Comment by novaugust** [comment]
Happy with this now. Hopefully the tests agree with me

**Comment by novaugust** [comment]
derp, ember 1.11.1 is out already, wipping

**Comment by ErisDS** [comment]
Does that mean the hack is gone? :wink:

**Comment by novaugust** [comment]
no more hack! Sorry, i updated the commit but not the pr message

**Comment by ErisDS** [comment]
Sorry, I updated my previous message and you wouldn't have seen the additional questions by email: 

> Second question, why the back ticks? I think this is something to do with ES6 but a short explanation/link to a resource would be lovely.

> Finally, this will need rebasing because I merged #5098 ;)

I've taken this PR for a very quick spin and it all seemed ok.

**Comment by novaugust** [comment]
The back ticks are string interpolation, es6 feature. I did them because...

```js
// I needed to build the entire string, then wrap that expression with parenthesis so that i could call a method on the result....
return ('background-image: url(' + this.get('foo') + ')').htmlSafe();
// equivalent, but more succinct eh
return `background-image: url(${this.get('foo')})`.htmlSafe();
```

**Comment by novaugust** [comment]
Updated

**Comment by novaugust** [comment]
And sorry, meant to make a link out of [string interpolation](http://babeljs.io/docs/learn-es6/#template-strings)

File: core/client/app/mixins/pagination-controller.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ var PaginationControllerMixin = Ember.Mixin.create({
 
         resetPagination: function () {
             this.set('paginationSettings.page', 1);
-            this.store.metaForType('tag', {pagination: undefined});
+            this.store.setMetaDataFor('tag', {pagination: undefined});
```
Comment (novaugust) [inline core/client/app/mixins/pagination-controller.js]: CLEARLY THIS CODE IS WRONG SIR.
GOOD DAY SIR.
:tophat:

Comment (JohnONolan) [inline core/client/app/mixins/pagination-controller.js]: :joy:

**Pull request merged**